docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,77 @@
+"""
+Coriolis parameters across the world
+====================================
+
+This code computes Coriolis parameters and stores them in a cube with
+associated metadata.
+
+It creates and plots a map showing the parameters, overlaid on top of an
```

Comment:
This reads oddly and uses incorrect terminology. Replace with:

```It plots the Coriolis frequency on an orthographic projection.```

I'd suggest that this line is part of the previous paragraph too.